test/static_a_token.test.ts

```diff
@@ -216,7 +224,7 @@ describe("static_a_token", function () {
         rewards_index: {
           wad: {
             high: 0,
-            low: BigInt(decimalToWad(2)),
+            low: BigInt(2),
```

Comment:
why not `rewards_index_value` here as well? we should have `2 * WAD` here as well, right?

Reply:
but the purpose here is to introduce a value which is inferior to 2*WAD and check if it will be rejected